### ISSUE TITLE
The Secret to Aging in Reverse | David Sinclair

### ISSUE THREAD

**Issue by nelsonic** [dwyl/health#114]
https://youtu.be/IEz1P4i1P7s
I've added this book to my cart/list:
Lifespan: Why We Age - and Why We Don't Have To https://www.amazon.co.uk/dp/0008353743/

But _first_ ... https://github.com/nelsonic/time-mvp-phoenix/issues/3

**Comment by iteles** [comment]
How is this dude 50? He clearly practices what he preaches!

**Comment by nelsonic** [comment]
Indeed ... 💭

**Comment by nelsonic** [comment]
David Sinclair - Cracking & reversing the aging clock - Science Unlimited 2019
https://youtu.be/Eu0GksWf5pI

> that talk was not very enlightening. It's very "high level" and has nothing _actionable_.

**Comment by nelsonic** [comment]
David Sinclair Is Extending Human Lifespan | Rich Roll Podcast https://youtu.be/bF5yl9OGKzw
[![image](https://user-images.githubusercontent.com/194400/65745467-b2072c00-e0f3-11e9-83a1-b0eb603d9d96.png)](https://youtu.be/bF5yl9OGKzw)

**Comment by iteles** [comment]
I listened to the Tom Bilyeu interview yesterday whilst cooking & reckon this is audible credit worthy. 
![AF3FB779-00BF-4F1D-8D05-1FBA80C8AF95](https://user-images.githubusercontent.com/4185328/65745237-f2b27580-e0f2-11e9-8844-a976b3bf5c65.png)

**Comment by nelsonic** [comment]
@iteles indeed. I was planning on buying the _physical_ book. (_it's already in the shopping cart_)
![image](https://user-images.githubusercontent.com/194400/65745380-681e4600-e0f3-11e9-904f-979b9f682286.png)

But given that we have Audible Credits and we (_you_) are far more likely to _listen_ to it for 11h than (_while doing other things_) than sit down and read (_for probably way more than 11h_) ... go for it.

The Rich Roll podcast episode is 2.2h and probably only has "teaser" / taster bits of info. 💭 
Might as well skip the podcast and jump straight into the original source material (_the audiobook_). 👍

**Comment by nelsonic** [comment]
Rewinding the clock on aging blood vessels https://youtu.be/Mf6bQZh_GYs
This quick intro to NAD anx SERT1 IS ONLY 2.4 mins and is way more informative.
